src/test/java/com/google/api/codegen/testsrc/library_gapic.yaml

```diff
@@ -647,9 +647,11 @@ interfaces:
       - names[0]=Odyssey
       - shelves[0]=Classics
       output_specs:
+        - assign:
+            books=$resp.names
```

Comment:
I like this more compact format! I'll update the doc.

My stylistic preference: I would put it in the same line when it fits, for compactness:
```
- assign: books=$resp.names
```

Reply:
Oh right, done